src/HooksWorkerClient.js

```diff
@@ -4,10 +4,10 @@ const path = require('path');
 const spawnArgs = require('spawn-args');
 const { EventEmitter } = require('events');
 
-const getGoBin = require('./get-go-bin');
+const getGoBin = require('./getGoBin');
```

Comment:
Maybe it would be nicer to rename this `getGoBinary`, it's a lot clearer on what it does.